### PR TITLE
Add ports forwarding script

### PR DESCRIPTION
**Pull request by osdnk**

### CODE REVIEW

**Comment by skylarbarrera** [comment]
what is this for? dev tools?

**Comment by osdnk** [comment]
yes

**Review by mikedemarais — APPROVED** [review]
lgtm

**Review by skylarbarrera — APPROVED** [review]
think we should call it android-devtools or something along those lines but otherwise I'm good with it 👍

**Comment by mikedemarais** [comment]
> think we should call it android-devtools or something along those lines but otherwise I'm good with it 👍

hmm, ya there could be a better name, this command only needs to be run once ya? should it be called `yarn setup-android`?
@skylarbarrera @osdnk

File: package.json [hunk 1]
```diff
@@ -28,6 +28,8 @@
     "start:clean": "watchman watch-del-all && rm -rf $TMPDIR/react-* && rm -rf $TMPDIR/metro-* && rm -rf $TMPDIR/haste-map-* && react-native start -- --reset-cache",
     "test": "jest test.js",
     "update-env": "curl -H 'Authorization: token '$GITHUB_TOKEN -H 'Accept: application/vnd.github.v3.raw' -O  -L https://api.github.com/repos/rainbow-me/rainbow-env/contents/dotenv && rm -f .env && mv dotenv .env",
+    "ganache": "source .env && ganache-cli --f https://mainnet.infura.io/v3/${INFURA_PROJECT_ID_DEV} -p 7545 -i 1 -b 10",
+    "ports-forward": "adb reverse tcp:8097 tcp:8097 && adb reverse tcp:8081 tcp:8081",
```
Comment (osdnk) [inline package.json:32]:
```suggestion
    "android-reverse-ports": "adb reverse tcp:8097 tcp:8097 && adb reverse tcp:8081 tcp:8081",
```

**Pull request merged**